src/redisearch_rs/inverted_index_bencher/benches/encoding_decoding.rs

```diff
@@ -26,5 +26,20 @@ fn benchmark_freqs_only(c: &mut Criterion) {
     bencher.decoding(c);
 }
 
-criterion_group!(benches, benchmark_numeric, benchmark_freqs_only);
+fn benchmark_freqs_fields(c: &mut Criterion) {
+    let bencher = benchers::freqs_fields::Bencher::new(false);
+    bencher.encoding(c);
+    bencher.decoding(c);
+
+    let bencher = benchers::freqs_fields::Bencher::new(true);
+    bencher.encoding(c);
+    bencher.decoding(c);
+}
```

Comment:
The `new(false|true)` is not intuitive at first glance. Maybe make a `default()` and `wide()` versions like earlier?